app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -14,69 +18,82 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
 import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.fasterxml.jackson.databind.node.JsonNodeFactory;
 import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.ser.FilterProvider;
+import com.fasterxml.jackson.databind.ser.impl.SimpleBeanPropertyFilter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
 
 @DynamoDBTable(tableName = "Study")
 @BridgeTypeName("Study")
+@JsonFilter("filter") 
 public class DynamoStudy implements Study {
-
-    // We need the unconfigured mapper for setData/getData.
+    
     private static ObjectMapper mapper = new ObjectMapper();
 
-    private static final String IDENTIFIER_PROPERTY = "identifier";
-    private static final String NAME_PROPERTY = "name";
-    private static final String MAX_NUM_OF_PARTICIPANTS_PROPERTY = "maxNumOfParticipants";
-    private static final String MIN_AGE_OF_CONSENT_PROPERTY = "minAgeOfConsent";
     private static final String RESEARCHER_ROLE_PROPERTY = "researcherRole";
     private static final String STORMPATH_HREF_PROPERTY = "stormpathHref";
-    private static final String VERSION_PROPERTY = "version";
+
+    private static final String MAX_NUM_OF_PARTICIPANTS_PROPERTY = "maxNumOfParticipants";
+    private static final String MIN_AGE_OF_CONSENT_PROPERTY = "minAgeOfConsent";
     private static final String SUPPORT_EMAIL_PROPERTY = "supportEmail";
     private static final String CONSENT_NOTIFICATION_EMAIL_PROPERTY = "consentNotificationEmail";
     private static final String USER_PROFILE_ATTRIBUTES_PROPERTY = "userProfileAttributes";
+    private static final String PASSWORD_POLICY_PROPERTY = "passwordPolicy";
+    private static final String VERIFY_EMAIL_TEMPLATE_PROPERTY = "verifyEmailTemplate";
+    private static final String RESET_PASSWORD_TEMPLATE_PROPERTY = "resetPasswordTemplate";
+    private static final String ACTIVE_PROPERTY = "active";
+    private static final String TECHNICAL_EMAIL_PROPERTY = "technicalEmail";
+    private static final String SPONSOR_NAME_PROPERTY = "sponsorName";
+
+    private static final FilterProvider RESEARCHER_VIEW_FILTER = new SimpleFilterProvider()
+        .addFilter("filter", SimpleBeanPropertyFilter.serializeAllExcept(STORMPATH_HREF_PROPERTY, RESEARCHER_ROLE_PROPERTY));
     
+    public static final ObjectWriter STUDY_WRITER = new BridgeObjectMapper().writer(
+        DynamoStudy.RESEARCHER_VIEW_FILTER);
+
     private String name;
+    private String sponsorName;
     private String identifier;
     private String researcherRole;
     private String stormpathHref;
     private String supportEmail;
+    private String technicalEmail;
     private String consentNotificationEmail;
     private int minAgeOfConsent;
     private int maxNumOfParticipants;
     private Long version;
+    private boolean active;
```

Comment:
This will default to false for studies that predate this flag. Are we gonna pre-emptively backfill all studies with this flag? Or should this be a Boolean so we can have null values?